src/HW_models/NRF_regs.h

```diff
@@ -1,8 +1,14 @@
-/*
+/**
  * Hacked version of nrf52.h (from the nrfx mdk)
  * without the base addresses, the includes and a few defines
  * to be used by the HW models
+ *
+ * Note that there is a few changes compared to the original
+ * Where relevant, these changes are:
+ * Copyright (c) 2017 Oticon A/S
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I think this note can only apply to the copyright statement, the license is the same for the whole file. Move the "SPDX-License-Identifier: BSD-3-Clause" line above the note to avoid the impression that the note tries to apply a different license to the changes than the rest of the file.

Reply:
Fair enough. Fixed :)